samples/examples/FSharpExamples/NaiveBayes.fs

```diff
@@ -1,4 +1,4 @@
-(*---------------------------------------------------------------------------
+﻿(*---------------------------------------------------------------------------
```

Comment:
Maybe adding a little bit more comments to the code?  Unlike wordcount or matchword that is very trivial, this one may not be that apparent to users without background.  Some comments on the over idea may help.